### ISSUE TITLE
cMDS should accept "metric" kwarg instead of "dissimilarity" 

### ISSUE THREAD

**Issue by bdpedigo** [neurodata/graspy#337]
https://scikit-learn.org/stable/modules/generated/sklearn.manifold.MDS.html
https://scikit-learn.org/stable/modules/generated/sklearn.manifold.Isomap.html
https://scikit-learn.org/stable/modules/generated/sklearn.manifold.TSNE.html

IMO for consistency, we should be compliant with the manifold learning stuff above, which all uses the term "metric" instead of "dissimilarity". Could accept both interchangeably if we really want.

**Comment by bdpedigo** [comment]
@j1c thoughts on this?

**Comment by bdpedigo** [comment]
whoops their MDS does accept dissimilarity. Do you know if this has real semantic meaning for them?
